src/controllers/chat.controller.js

```diff
@@ -0,0 +1,31 @@
+import { Socket } from "socket.io";
+import models from "../database/models";
+import formatMessage from "../helpers/messageHelper";
+
+const chat = async(socket) => {
+    socket.on("message", async(msg) => {
+        const { userName, id} = socket.data.user;
+        models.Chat.create({content: msg, userId: id}).then(() => {
+            console.log(`New message received: ${msg} by username ${userName}`);
+            socket.broadcast.emit('message', formatMessage(userName,msg));
+        })
+    })
+}
+
+const getChat = async(req,res) => {
+    try {
+        if (req.query.df) {
+            throw new Error("No Parameters required");
+        }
+        const chats = await models.Chat.findAll()
+        return res.status(200).json({"Chats": chats});
+    } catch (error) {
+        console.log(error)
```

Comment:
remove `console.log`

Reply:
Ooohhh, i was using this to read errors on the console. Sorry I forgot to change it.